2023-01-09T04:51:40.502311835Z I0109 04:51:40.502206       1 flags.go:64] FLAG: --accesstoken-inactivity-timeout="0s"
2023-01-09T04:51:40.502311835Z I0109 04:51:40.502276       1 flags.go:64] FLAG: --add-dir-header="false"
2023-01-09T04:51:40.502311835Z I0109 04:51:40.502281       1 flags.go:64] FLAG: --admission-control-config-file=""
2023-01-09T04:51:40.502311835Z I0109 04:51:40.502286       1 flags.go:64] FLAG: --advertise-address="<nil>"
2023-01-09T04:51:40.502311835Z I0109 04:51:40.502290       1 flags.go:64] FLAG: --alsologtostderr="false"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502294       1 flags.go:64] FLAG: --api-audiences="[https://kubernetes.default.svc]"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502300       1 flags.go:64] FLAG: --audit-log-batch-buffer-size="10000"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502304       1 flags.go:64] FLAG: --audit-log-batch-max-size="1"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502307       1 flags.go:64] FLAG: --audit-log-batch-max-wait="0s"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502310       1 flags.go:64] FLAG: --audit-log-batch-throttle-burst="0"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502313       1 flags.go:64] FLAG: --audit-log-batch-throttle-enable="false"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502317       1 flags.go:64] FLAG: --audit-log-batch-throttle-qps="0"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502320       1 flags.go:64] FLAG: --audit-log-compress="false"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502323       1 flags.go:64] FLAG: --audit-log-format="json"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502326       1 flags.go:64] FLAG: --audit-log-maxage="0"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502330       1 flags.go:64] FLAG: --audit-log-maxbackup="10"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502332       1 flags.go:64] FLAG: --audit-log-maxsize="100"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502335       1 flags.go:64] FLAG: --audit-log-mode="blocking"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502339       1 flags.go:64] FLAG: --audit-log-path="/var/log/oauth-apiserver/audit.log"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502342       1 flags.go:64] FLAG: --audit-log-truncate-enabled="false"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502345       1 flags.go:64] FLAG: --audit-log-truncate-max-batch-size="10485760"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502349       1 flags.go:64] FLAG: --audit-log-truncate-max-event-size="102400"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502352       1 flags.go:64] FLAG: --audit-log-version="audit.k8s.io/v1"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502355       1 flags.go:64] FLAG: --audit-policy-file="/var/run/configmaps/audit/policy.yaml"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502358       1 flags.go:64] FLAG: --audit-webhook-batch-buffer-size="10000"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502362       1 flags.go:64] FLAG: --audit-webhook-batch-initial-backoff="10s"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502365       1 flags.go:64] FLAG: --audit-webhook-batch-max-size="400"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502367       1 flags.go:64] FLAG: --audit-webhook-batch-max-wait="30s"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502370       1 flags.go:64] FLAG: --audit-webhook-batch-throttle-burst="15"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502373       1 flags.go:64] FLAG: --audit-webhook-batch-throttle-enable="true"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502376       1 flags.go:64] FLAG: --audit-webhook-batch-throttle-qps="10"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502379       1 flags.go:64] FLAG: --audit-webhook-config-file=""
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502382       1 flags.go:64] FLAG: --audit-webhook-initial-backoff="10s"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502385       1 flags.go:64] FLAG: --audit-webhook-mode="batch"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502388       1 flags.go:64] FLAG: --audit-webhook-truncate-enabled="false"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502390       1 flags.go:64] FLAG: --audit-webhook-truncate-max-batch-size="10485760"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502393       1 flags.go:64] FLAG: --audit-webhook-truncate-max-event-size="102400"
2023-01-09T04:51:40.502402164Z I0109 04:51:40.502396       1 flags.go:64] FLAG: --audit-webhook-version="audit.k8s.io/v1"
2023-01-09T04:51:40.502420396Z I0109 04:51:40.502399       1 flags.go:64] FLAG: --authentication-kubeconfig=""
2023-01-09T04:51:40.502420396Z I0109 04:51:40.502403       1 flags.go:64] FLAG: --authentication-skip-lookup="false"
2023-01-09T04:51:40.502420396Z I0109 04:51:40.502406       1 flags.go:64] FLAG: --authentication-token-webhook-cache-ttl="10s"
2023-01-09T04:51:40.502420396Z I0109 04:51:40.502408       1 flags.go:64] FLAG: --authentication-tolerate-lookup-failure="false"
2023-01-09T04:51:40.502420396Z I0109 04:51:40.502411       1 flags.go:64] FLAG: --authorization-always-allow-paths="[/healthz,/readyz,/livez]"
2023-01-09T04:51:40.502420396Z I0109 04:51:40.502415       1 flags.go:64] FLAG: --authorization-kubeconfig=""
2023-01-09T04:51:40.502429890Z I0109 04:51:40.502418       1 flags.go:64] FLAG: --authorization-webhook-cache-authorized-ttl="10s"
2023-01-09T04:51:40.502429890Z I0109 04:51:40.502421       1 flags.go:64] FLAG: --authorization-webhook-cache-unauthorized-ttl="10s"
2023-01-09T04:51:40.502429890Z I0109 04:51:40.502424       1 flags.go:64] FLAG: --bind-address="0.0.0.0"
2023-01-09T04:51:40.502437801Z I0109 04:51:40.502427       1 flags.go:64] FLAG: --cert-dir="apiserver.local.config/certificates"
2023-01-09T04:51:40.502437801Z I0109 04:51:40.502431       1 flags.go:64] FLAG: --client-ca-file=""
2023-01-09T04:51:40.502437801Z I0109 04:51:40.502433       1 flags.go:64] FLAG: --contention-profiling="false"
2023-01-09T04:51:40.502445492Z I0109 04:51:40.502436       1 flags.go:64] FLAG: --cors-allowed-origins="[//127\\.0\\.0\\.1(:|$),//localhost(:|$)]"
2023-01-09T04:51:40.502445492Z I0109 04:51:40.502441       1 flags.go:64] FLAG: --default-watch-cache-size="100"
2023-01-09T04:51:40.502452995Z I0109 04:51:40.502444       1 flags.go:64] FLAG: --delete-collection-workers="1"
2023-01-09T04:51:40.502452995Z I0109 04:51:40.502446       1 flags.go:64] FLAG: --disable-admission-plugins="[]"
2023-01-09T04:51:40.502452995Z I0109 04:51:40.502450       1 flags.go:64] FLAG: --egress-selector-config-file=""
2023-01-09T04:51:40.502460618Z I0109 04:51:40.502453       1 flags.go:64] FLAG: --enable-admission-plugins="[]"
2023-01-09T04:51:40.502460618Z I0109 04:51:40.502457       1 flags.go:64] FLAG: --enable-garbage-collector="true"
2023-01-09T04:51:40.502468027Z I0109 04:51:40.502460       1 flags.go:64] FLAG: --enable-priority-and-fairness="true"
2023-01-09T04:51:40.502468027Z I0109 04:51:40.502463       1 flags.go:64] FLAG: --encryption-provider-config=""
2023-01-09T04:51:40.502475365Z I0109 04:51:40.502466       1 flags.go:64] FLAG: --etcd-cafile="/var/run/configmaps/etcd-serving-ca/ca-bundle.crt"
2023-01-09T04:51:40.502475365Z I0109 04:51:40.502469       1 flags.go:64] FLAG: --etcd-certfile="/var/run/secrets/etcd-client/tls.crt"
2023-01-09T04:51:40.502475365Z I0109 04:51:40.502472       1 flags.go:64] FLAG: --etcd-compaction-interval="5m0s"
2023-01-09T04:51:40.502485428Z I0109 04:51:40.502475       1 flags.go:64] FLAG: --etcd-count-metric-poll-period="1m0s"
2023-01-09T04:51:40.502485428Z I0109 04:51:40.502478       1 flags.go:64] FLAG: --etcd-db-metric-poll-interval="30s"
2023-01-09T04:51:40.502485428Z I0109 04:51:40.502481       1 flags.go:64] FLAG: --etcd-healthcheck-timeout="10s"
2023-01-09T04:51:40.502493246Z I0109 04:51:40.502484       1 flags.go:64] FLAG: --etcd-keyfile="/var/run/secrets/etcd-client/tls.key"
2023-01-09T04:51:40.502493246Z I0109 04:51:40.502487       1 flags.go:64] FLAG: --etcd-prefix="openshift.io"
2023-01-09T04:51:40.502500676Z I0109 04:51:40.502490       1 flags.go:64] FLAG: --etcd-servers="[https://10.0.145.4:2379,https://10.0.160.211:2379,https://10.0.199.219:2379]"
2023-01-09T04:51:40.502500676Z I0109 04:51:40.502495       1 flags.go:64] FLAG: --etcd-servers-overrides="[]"
2023-01-09T04:51:40.502508022Z I0109 04:51:40.502499       1 flags.go:64] FLAG: --external-hostname=""
2023-01-09T04:51:40.502508022Z I0109 04:51:40.502502       1 flags.go:64] FLAG: --feature-gates=""
2023-01-09T04:51:40.502515690Z I0109 04:51:40.502507       1 flags.go:64] FLAG: --goaway-chance="0"
2023-01-09T04:51:40.502515690Z I0109 04:51:40.502510       1 flags.go:64] FLAG: --help="false"
2023-01-09T04:51:40.502515690Z I0109 04:51:40.502513       1 flags.go:64] FLAG: --http2-max-streams-per-connection="1000"
2023-01-09T04:51:40.502523277Z I0109 04:51:40.502516       1 flags.go:64] FLAG: --kubeconfig=""
2023-01-09T04:51:40.502523277Z I0109 04:51:40.502519       1 flags.go:64] FLAG: --lease-reuse-duration-seconds="60"
2023-01-09T04:51:40.502530589Z I0109 04:51:40.502522       1 flags.go:64] FLAG: --livez-grace-period="0s"
2023-01-09T04:51:40.502530589Z I0109 04:51:40.502525       1 flags.go:64] FLAG: --log-backtrace-at=":0"
2023-01-09T04:51:40.502537993Z I0109 04:51:40.502530       1 flags.go:64] FLAG: --log-dir=""
2023-01-09T04:51:40.502537993Z I0109 04:51:40.502533       1 flags.go:64] FLAG: --log-file=""
2023-01-09T04:51:40.502545455Z I0109 04:51:40.502536       1 flags.go:64] FLAG: --log-file-max-size="1800"
2023-01-09T04:51:40.502545455Z I0109 04:51:40.502539       1 flags.go:64] FLAG: --log-flush-frequency="5s"
2023-01-09T04:51:40.502545455Z I0109 04:51:40.502542       1 flags.go:64] FLAG: --logtostderr="true"
2023-01-09T04:51:40.502553167Z I0109 04:51:40.502545       1 flags.go:64] FLAG: --master-service-namespace="default"
2023-01-09T04:51:40.502553167Z I0109 04:51:40.502548       1 flags.go:64] FLAG: --max-mutating-requests-inflight="200"
2023-01-09T04:51:40.502560490Z I0109 04:51:40.502551       1 flags.go:64] FLAG: --max-requests-inflight="400"
2023-01-09T04:51:40.502560490Z I0109 04:51:40.502554       1 flags.go:64] FLAG: --min-request-timeout="1800"
2023-01-09T04:51:40.502560490Z I0109 04:51:40.502557       1 flags.go:64] FLAG: --one-output="false"
2023-01-09T04:51:40.502568168Z I0109 04:51:40.502560       1 flags.go:64] FLAG: --permit-address-sharing="false"
2023-01-09T04:51:40.502568168Z I0109 04:51:40.502563       1 flags.go:64] FLAG: --permit-port-sharing="false"
2023-01-09T04:51:40.502575912Z I0109 04:51:40.502566       1 flags.go:64] FLAG: --profiling="true"
2023-01-09T04:51:40.502575912Z I0109 04:51:40.502569       1 flags.go:64] FLAG: --request-timeout="1m0s"
2023-01-09T04:51:40.502575912Z I0109 04:51:40.502573       1 flags.go:64] FLAG: --requestheader-allowed-names="[]"
2023-01-09T04:51:40.502583466Z I0109 04:51:40.502576       1 flags.go:64] FLAG: --requestheader-client-ca-file=""
2023-01-09T04:51:40.502583466Z I0109 04:51:40.502579       1 flags.go:64] FLAG: --requestheader-extra-headers-prefix="[x-remote-extra-]"
2023-01-09T04:51:40.502592889Z I0109 04:51:40.502583       1 flags.go:64] FLAG: --requestheader-group-headers="[x-remote-group]"
2023-01-09T04:51:40.502592889Z I0109 04:51:40.502587       1 flags.go:64] FLAG: --requestheader-username-headers="[x-remote-user]"
2023-01-09T04:51:40.502600404Z I0109 04:51:40.502592       1 flags.go:64] FLAG: --secure-port="8443"
2023-01-09T04:51:40.502600404Z I0109 04:51:40.502595       1 flags.go:64] FLAG: --shutdown-delay-duration="15s"
2023-01-09T04:51:40.502607700Z I0109 04:51:40.502598       1 flags.go:64] FLAG: --shutdown-send-retry-after="false"
2023-01-09T04:51:40.502607700Z I0109 04:51:40.502601       1 flags.go:64] FLAG: --skip-headers="false"
2023-01-09T04:51:40.502607700Z I0109 04:51:40.502604       1 flags.go:64] FLAG: --skip-log-headers="false"
2023-01-09T04:51:40.502615307Z I0109 04:51:40.502607       1 flags.go:64] FLAG: --stderrthreshold="2"
2023-01-09T04:51:40.502615307Z I0109 04:51:40.502610       1 flags.go:64] FLAG: --storage-backend=""
2023-01-09T04:51:40.502615307Z I0109 04:51:40.502612       1 flags.go:64] FLAG: --storage-media-type="application/json"
2023-01-09T04:51:40.502622918Z I0109 04:51:40.502615       1 flags.go:64] FLAG: --strict-transport-security-directives="[]"
2023-01-09T04:51:40.502622918Z I0109 04:51:40.502619       1 flags.go:64] FLAG: --tls-cert-file="/var/run/secrets/serving-cert/tls.crt"
2023-01-09T04:51:40.502632167Z I0109 04:51:40.502622       1 flags.go:64] FLAG: --tls-cipher-suites="[TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256]"
2023-01-09T04:51:40.502639591Z I0109 04:51:40.502631       1 flags.go:64] FLAG: --tls-min-version="VersionTLS12"
2023-01-09T04:51:40.502639591Z I0109 04:51:40.502634       1 flags.go:64] FLAG: --tls-private-key-file="/var/run/secrets/serving-cert/tls.key"
2023-01-09T04:51:40.502647004Z I0109 04:51:40.502637       1 flags.go:64] FLAG: --tls-sni-cert-key="[]"
2023-01-09T04:51:40.502647004Z I0109 04:51:40.502642       1 flags.go:64] FLAG: --tracing-config-file=""
2023-01-09T04:51:40.502654293Z I0109 04:51:40.502645       1 flags.go:64] FLAG: --v="2"
2023-01-09T04:51:40.502654293Z I0109 04:51:40.502648       1 flags.go:64] FLAG: --vmodule=""
2023-01-09T04:51:40.502654293Z I0109 04:51:40.502651       1 flags.go:64] FLAG: --watch-cache="true"
2023-01-09T04:51:40.502661923Z I0109 04:51:40.502654       1 flags.go:64] FLAG: --watch-cache-sizes="[]"
2023-01-09T04:51:40.503488343Z I0109 04:51:40.503460       1 dynamic_serving_content.go:113] "Loaded a new cert/key pair" name="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key"
2023-01-09T04:51:40.723647461Z I0109 04:51:40.723605       1 requestheader_controller.go:244] Loaded a new request header values for RequestHeaderAuthRequestController
2023-01-09T04:51:40.724570569Z I0109 04:51:40.724543       1 audit.go:349] Using audit backend: ignoreErrors<log>
2023-01-09T04:51:40.725247011Z I0109 04:51:40.725229       1 plugins.go:158] Loaded 2 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,MutatingAdmissionWebhook.
2023-01-09T04:51:40.725247011Z I0109 04:51:40.725243       1 plugins.go:161] Loaded 1 validating admission controller(s) successfully in the following order: ValidatingAdmissionWebhook.
2023-01-09T04:51:40.725386837Z I0109 04:51:40.725368       1 apf_controller.go:218] NewTestableController "Controller" with serverConcurrencyLimit=600, requestWaitLimit=15s, name=Controller, asFieldManager="api-priority-and-fairness-config-consumer-v1"
2023-01-09T04:51:40.746472372Z I0109 04:51:40.746431       1 store.go:1452] "Monitoring resource count at path" resource="oauthclients.oauth.openshift.io" path="<storage-prefix>//oauth/clients"
2023-01-09T04:51:40.751882291Z I0109 04:51:40.751830       1 cacher.go:410] cacher (*oauth.OAuthClient): initialized
2023-01-09T04:51:40.759254137Z I0109 04:51:40.759214       1 store.go:1452] "Monitoring resource count at path" resource="oauthauthorizetokens.oauth.openshift.io" path="<storage-prefix>//oauth/authorizetokens"
2023-01-09T04:51:40.760626562Z I0109 04:51:40.760585       1 cacher.go:410] cacher (*oauth.OAuthAuthorizeToken): initialized
2023-01-09T04:51:40.765701119Z I0109 04:51:40.765674       1 store.go:1452] "Monitoring resource count at path" resource="oauthaccesstokens.oauth.openshift.io" path="<storage-prefix>//oauth/accesstokens"
2023-01-09T04:51:40.767053258Z I0109 04:51:40.766954       1 cacher.go:410] cacher (*oauth.OAuthAccessToken): initialized
2023-01-09T04:51:40.774325322Z I0109 04:51:40.774298       1 store.go:1452] "Monitoring resource count at path" resource="oauthclientauthorizations.oauth.openshift.io" path="<storage-prefix>//oauth/clientauthorizations"
2023-01-09T04:51:40.775671197Z I0109 04:51:40.775643       1 cacher.go:410] cacher (*oauth.OAuthClientAuthorization): initialized
2023-01-09T04:51:40.802967555Z I0109 04:51:40.802920       1 store.go:1452] "Monitoring resource count at path" resource="users.user.openshift.io" path="<storage-prefix>//users"
2023-01-09T04:51:40.804431384Z I0109 04:51:40.804402       1 cacher.go:410] cacher (*user.User): initialized
2023-01-09T04:51:40.809312824Z I0109 04:51:40.809284       1 store.go:1452] "Monitoring resource count at path" resource="identities.user.openshift.io" path="<storage-prefix>//useridentities"
2023-01-09T04:51:40.810975679Z I0109 04:51:40.810950       1 cacher.go:410] cacher (*user.Identity): initialized
2023-01-09T04:51:40.817842640Z I0109 04:51:40.817820       1 store.go:1452] "Monitoring resource count at path" resource="groups.user.openshift.io" path="<storage-prefix>//groups"
2023-01-09T04:51:40.819130497Z I0109 04:51:40.819104       1 cacher.go:410] cacher (*user.Group): initialized
2023-01-09T04:51:41.015728750Z I0109 04:51:41.015688       1 aggregator.go:115] Building initial OpenAPI spec
2023-01-09T04:51:41.018097022Z I0109 04:51:41.018071       1 aggregator.go:118] Finished initial OpenAPI spec generation after 2.358835ms
2023-01-09T04:51:41.018211133Z I0109 04:51:41.018187       1 genericapiserver.go:418] MuxAndDiscoveryComplete has all endpoints registered and discovery information is complete
2023-01-09T04:51:41.021435853Z I0109 04:51:41.021402       1 requestheader_controller.go:169] Starting RequestHeaderAuthRequestController
2023-01-09T04:51:41.021435853Z I0109 04:51:41.021412       1 configmap_cafile_content.go:202] "Starting controller" name="client-ca::kube-system::extension-apiserver-authentication::client-ca-file"
2023-01-09T04:51:41.021435853Z I0109 04:51:41.021426       1 configmap_cafile_content.go:202] "Starting controller" name="client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file"
2023-01-09T04:51:41.021479119Z I0109 04:51:41.021437       1 shared_informer.go:255] Waiting for caches to sync for client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file
2023-01-09T04:51:41.021479119Z I0109 04:51:41.021428       1 shared_informer.go:255] Waiting for caches to sync for client-ca::kube-system::extension-apiserver-authentication::client-ca-file
2023-01-09T04:51:41.021544058Z I0109 04:51:41.021430       1 shared_informer.go:255] Waiting for caches to sync for RequestHeaderAuthRequestController
2023-01-09T04:51:41.021724214Z I0109 04:51:41.021698       1 dynamic_serving_content.go:132] "Starting controller" name="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key"
2023-01-09T04:51:41.021750853Z I0109 04:51:41.021732       1 tlsconfig.go:200] "Loaded serving cert" certName="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key" certDetail="\"api.openshift-oauth-apiserver.svc\" [serving] validServingFor=[api.openshift-oauth-apiserver.svc,api.openshift-oauth-apiserver.svc.cluster.local] issuer=\"openshift-service-serving-signer@1673239264\" (2023-01-09 04:41:24 +0000 UTC to 2025-01-08 04:41:25 +0000 UTC (now=2023-01-09 04:51:41.021705355 +0000 UTC))"
2023-01-09T04:51:41.021886604Z I0109 04:51:41.021869       1 named_certificates.go:53] "Loaded SNI cert" index=0 certName="self-signed loopback" certDetail="\"apiserver-loopback-client@1673239900\" [serving] validServingFor=[apiserver-loopback-client] issuer=\"apiserver-loopback-client-ca@1673239900\" (2023-01-09 03:51:40 +0000 UTC to 2024-01-09 03:51:40 +0000 UTC (now=2023-01-09 04:51:41.021852041 +0000 UTC))"
2023-01-09T04:51:41.021909460Z I0109 04:51:41.021899       1 secure_serving.go:210] Serving securely on [::]:8443
2023-01-09T04:51:41.021936339Z I0109 04:51:41.021922       1 genericapiserver.go:488] [graceful-termination] waiting for shutdown to be initiated
2023-01-09T04:51:41.021958173Z I0109 04:51:41.021944       1 tlsconfig.go:240] "Starting DynamicServingCertificateController"
2023-01-09T04:51:41.022047164Z I0109 04:51:41.022027       1 apf_controller.go:317] Starting API Priority and Fairness config controller
2023-01-09T04:51:41.121618344Z I0109 04:51:41.121575       1 shared_informer.go:262] Caches are synced for client-ca::kube-system::extension-apiserver-authentication::client-ca-file
2023-01-09T04:51:41.121666349Z I0109 04:51:41.121579       1 shared_informer.go:262] Caches are synced for client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file
2023-01-09T04:51:41.121723036Z I0109 04:51:41.121707       1 shared_informer.go:262] Caches are synced for RequestHeaderAuthRequestController
2023-01-09T04:51:41.121830556Z I0109 04:51:41.121815       1 tlsconfig.go:178] "Loaded client CA" index=0 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"aggregator-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:22 +0000 UTC to 2023-01-10 04:28:22 +0000 UTC (now=2023-01-09 04:51:41.121790956 +0000 UTC))"
2023-01-09T04:51:41.122014157Z I0109 04:51:41.121976       1 tlsconfig.go:200] "Loaded serving cert" certName="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key" certDetail="\"api.openshift-oauth-apiserver.svc\" [serving] validServingFor=[api.openshift-oauth-apiserver.svc,api.openshift-oauth-apiserver.svc.cluster.local] issuer=\"openshift-service-serving-signer@1673239264\" (2023-01-09 04:41:24 +0000 UTC to 2025-01-08 04:41:25 +0000 UTC (now=2023-01-09 04:51:41.121958893 +0000 UTC))"
2023-01-09T04:51:41.122127395Z I0109 04:51:41.122114       1 named_certificates.go:53] "Loaded SNI cert" index=0 certName="self-signed loopback" certDetail="\"apiserver-loopback-client@1673239900\" [serving] validServingFor=[apiserver-loopback-client] issuer=\"apiserver-loopback-client-ca@1673239900\" (2023-01-09 03:51:40 +0000 UTC to 2024-01-09 03:51:40 +0000 UTC (now=2023-01-09 04:51:41.122099073 +0000 UTC))"
2023-01-09T04:51:41.122267858Z I0109 04:51:41.122255       1 tlsconfig.go:178] "Loaded client CA" index=0 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"admin-kubeconfig-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:21 +0000 UTC to 2033-01-06 04:28:21 +0000 UTC (now=2023-01-09 04:51:41.122241613 +0000 UTC))"
2023-01-09T04:51:41.122286949Z I0109 04:51:41.122278       1 tlsconfig.go:178] "Loaded client CA" index=1 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kubelet-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:24 +0000 UTC to 2023-01-10 04:28:24 +0000 UTC (now=2023-01-09 04:51:41.122268094 +0000 UTC))"
2023-01-09T04:51:41.122311554Z I0109 04:51:41.122299       1 tlsconfig.go:178] "Loaded client CA" index=2 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kube-control-plane-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:24 +0000 UTC to 2024-01-09 04:28:24 +0000 UTC (now=2023-01-09 04:51:41.122285212 +0000 UTC))"
2023-01-09T04:51:41.122324208Z I0109 04:51:41.122318       1 tlsconfig.go:178] "Loaded client CA" index=3 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kube-apiserver-to-kubelet-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:24 +0000 UTC to 2024-01-09 04:28:24 +0000 UTC (now=2023-01-09 04:51:41.122309075 +0000 UTC))"
2023-01-09T04:51:41.122347361Z I0109 04:51:41.122335       1 tlsconfig.go:178] "Loaded client CA" index=4 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kubelet-bootstrap-kubeconfig-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:22 +0000 UTC to 2033-01-06 04:28:22 +0000 UTC (now=2023-01-09 04:51:41.122324857 +0000 UTC))"
2023-01-09T04:51:41.122365683Z I0109 04:51:41.122355       1 tlsconfig.go:178] "Loaded client CA" index=5 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kube-csr-signer_@1673239265\" [] issuer=\"kubelet-signer\" (2023-01-09 04:41:04 +0000 UTC to 2023-01-10 04:28:24 +0000 UTC (now=2023-01-09 04:51:41.122345032 +0000 UTC))"
2023-01-09T04:51:41.122387708Z I0109 04:51:41.122375       1 tlsconfig.go:178] "Loaded client CA" index=6 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"openshift-kube-apiserver-operator_node-system-admin-signer@1673239266\" [] issuer=\"<self>\" (2023-01-09 04:41:06 +0000 UTC to 2024-01-09 04:41:07 +0000 UTC (now=2023-01-09 04:51:41.122363678 +0000 UTC))"
2023-01-09T04:51:41.122399829Z I0109 04:51:41.122394       1 tlsconfig.go:178] "Loaded client CA" index=7 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"aggregator-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:22 +0000 UTC to 2023-01-10 04:28:22 +0000 UTC (now=2023-01-09 04:51:41.122384394 +0000 UTC))"
2023-01-09T04:51:41.122511143Z I0109 04:51:41.122497       1 tlsconfig.go:200] "Loaded serving cert" certName="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key" certDetail="\"api.openshift-oauth-apiserver.svc\" [serving] validServingFor=[api.openshift-oauth-apiserver.svc,api.openshift-oauth-apiserver.svc.cluster.local] issuer=\"openshift-service-serving-signer@1673239264\" (2023-01-09 04:41:24 +0000 UTC to 2025-01-08 04:41:25 +0000 UTC (now=2023-01-09 04:51:41.122483551 +0000 UTC))"
2023-01-09T04:51:41.122625698Z I0109 04:51:41.122596       1 named_certificates.go:53] "Loaded SNI cert" index=0 certName="self-signed loopback" certDetail="\"apiserver-loopback-client@1673239900\" [serving] validServingFor=[apiserver-loopback-client] issuer=\"apiserver-loopback-client-ca@1673239900\" (2023-01-09 03:51:40 +0000 UTC to 2024-01-09 03:51:40 +0000 UTC (now=2023-01-09 04:51:41.122584908 +0000 UTC))"
2023-01-09T04:51:41.122659819Z I0109 04:51:41.122626       1 apf_controller.go:322] Running API Priority and Fairness config worker
2023-01-09T04:52:01.528187439Z W0109 04:52:01.528145       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.528335001Z W0109 04:52:01.528317       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.528649141Z W0109 04:52:01.528624       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.528746023Z W0109 04:52:01.528628       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.528935996Z W0109 04:52:01.528908       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.529864380Z W0109 04:52:01.529805       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.530161930Z W0109 04:52:01.530138       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.533342548Z W0109 04:52:01.533320       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:01.533549262Z W0109 04:52:01.533530       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.528899031Z W0109 04:52:02.528860       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.528899031Z W0109 04:52:02.528870       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.528976337Z W0109 04:52:02.528907       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.528976337Z W0109 04:52:02.528967       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.529175964Z W0109 04:52:02.529158       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.530442349Z W0109 04:52:02.530407       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.530464935Z W0109 04:52:02.530438       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.533715351Z W0109 04:52:02.533689       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:02.533753987Z W0109 04:52:02.533738       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:03.867284160Z W0109 04:52:03.867233       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.029033340Z W0109 04:52:04.028966       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.094668244Z W0109 04:52:04.094619       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.150873415Z W0109 04:52:04.150835       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.175349589Z W0109 04:52:04.175310       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.219877242Z W0109 04:52:04.219834       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.245500409Z W0109 04:52:04.245463       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.273839895Z W0109 04:52:04.273789       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:04.379340058Z W0109 04:52:04.379295       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.210946067Z W0109 04:52:06.210904       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.213202744Z W0109 04:52:06.213177       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.331152030Z W0109 04:52:06.331113       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.363465261Z W0109 04:52:06.363417       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.772857235Z W0109 04:52:06.772817       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.841320945Z W0109 04:52:06.841281       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.982405054Z W0109 04:52:06.982365       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:06.982443578Z W0109 04:52:06.982375       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:07.098319719Z W0109 04:52:07.098281       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:09.729313697Z W0109 04:52:09.729270       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:10.129381977Z W0109 04:52:10.129339       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:10.541218476Z W0109 04:52:10.541176       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:10.669815617Z W0109 04:52:10.669768       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:10.831407303Z W0109 04:52:10.831347       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:10.848719843Z W0109 04:52:10.848677       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:11.201718155Z W0109 04:52:11.201673       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:11.220067992Z W0109 04:52:11.220033       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:11.220146532Z W0109 04:52:11.220115       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:11.689628102Z W0109 04:52:11.689549       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:12.690249738Z W0109 04:52:12.690209       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:14.059162135Z W0109 04:52:14.059117       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:16.144804209Z W0109 04:52:16.144767       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:16.448097408Z W0109 04:52:16.448051       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:16.650958647Z W0109 04:52:16.650912       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:16.661328453Z W0109 04:52:16.661288       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:16.673619275Z W0109 04:52:16.673584       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:16.950369734Z W0109 04:52:16.950316       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:17.215719024Z W0109 04:52:17.215678       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:17.787040116Z W0109 04:52:17.782384       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:18.149133224Z W0109 04:52:18.149092       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:18.336963463Z W0109 04:52:18.336920       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:20.941252016Z W0109 04:52:20.941212       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:24.923936154Z W0109 04:52:24.923889       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:25.207255067Z W0109 04:52:25.207212       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:25.724215751Z W0109 04:52:25.724159       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:25.990978380Z W0109 04:52:25.990933       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:26.028646182Z W0109 04:52:26.028611       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:26.597983442Z W0109 04:52:26.597945       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:26.619301448Z W0109 04:52:26.619274       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:26.814009192Z W0109 04:52:26.813955       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:26.914448388Z W0109 04:52:26.914410       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:28.064317981Z W0109 04:52:28.064278       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:40.581882517Z W0109 04:52:40.581842       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:42.270764105Z W0109 04:52:42.270719       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:42.933670851Z W0109 04:52:42.933630       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:43.717513847Z W0109 04:52:43.717474       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:43.858480075Z W0109 04:52:43.858440       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:44.438812286Z W0109 04:52:44.438769       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:45.487362985Z W0109 04:52:45.487323       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:45.859108214Z W0109 04:52:45.859054       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:52:46.588481753Z W0109 04:52:46.588435       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.597729110Z W0109 04:53:57.597629       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.597729110Z W0109 04:53:57.597695       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.597779638Z W0109 04:53:57.597739       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.597779638Z W0109 04:53:57.597756       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.598209532Z W0109 04:53:57.597853       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.598209532Z W0109 04:53:57.597869       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.598209532Z W0109 04:53:57.597905       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.598209532Z W0109 04:53:57.597930       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.598609806Z W0109 04:53:57.598587       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599598193Z W0109 04:53:58.599539       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599628066Z W0109 04:53:58.599602       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599685665Z W0109 04:53:58.599636       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599724323Z W0109 04:53:58.599687       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599737139Z W0109 04:53:58.599722       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599787195Z W0109 04:53:58.599773       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599834975Z W0109 04:53:58.599815       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599856053Z W0109 04:53:58.599815       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.599856053Z W0109 04:53:58.599849       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:59.905583529Z W0109 04:53:59.905545       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:59.919545980Z W0109 04:53:59.919512       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:59.943519878Z W0109 04:53:59.943487       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:59.973479244Z W0109 04:53:59.973444       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.113360264Z W0109 04:54:00.113322       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.180617945Z W0109 04:54:00.180581       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.291118430Z W0109 04:54:00.291082       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.322062532Z W0109 04:54:00.322016       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.510463331Z W0109 04:54:00.510426       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.233287473Z W0109 04:54:02.233243       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.404691160Z W0109 04:54:02.404653       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.609561662Z W0109 04:54:02.609519       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.652301000Z W0109 04:54:02.652260       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.754536880Z W0109 04:54:02.754501       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.897629660Z W0109 04:54:02.897592       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.919621326Z W0109 04:54:02.919583       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:03.041116694Z W0109 04:54:03.041069       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:03.388058578Z W0109 04:54:03.388010       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:05.837417394Z W0109 04:54:05.837373       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.171350546Z W0109 04:54:06.171304       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.251639225Z W0109 04:54:06.251598       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.564389509Z W0109 04:54:06.564338       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.856036415Z W0109 04:54:06.855960       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:07.124102946Z W0109 04:54:07.124065       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:07.319780268Z W0109 04:54:07.319740       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:07.539860200Z W0109 04:54:07.539819       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:07.716032585Z W0109 04:54:07.715978       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:08.106569056Z W0109 04:54:08.106527       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:09.108041637Z W0109 04:54:09.107981       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:10.794022275Z W0109 04:54:10.793976       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:12.449541604Z W0109 04:54:12.449503       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:12.563110861Z W0109 04:54:12.563077       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:13.104056507Z W0109 04:54:13.104001       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:13.201906196Z W0109 04:54:13.201863       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:13.273247022Z W0109 04:54:13.273207       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:13.659451885Z W0109 04:54:13.659412       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:14.235334383Z W0109 04:54:14.235289       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:14.319377471Z W0109 04:54:14.319336       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:15.323585827Z W0109 04:54:15.323544       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:15.369604702Z W0109 04:54:15.369563       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:17.877810926Z W0109 04:54:17.877775       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:22.197606103Z W0109 04:54:22.197561       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:22.670951072Z W0109 04:54:22.670914       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:23.663120526Z W0109 04:54:23.663080       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:24.010034483Z W0109 04:54:24.009974       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:24.100493559Z W0109 04:54:24.100450       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:24.555906743Z W0109 04:54:24.555864       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:24.973251724Z W0109 04:54:24.973211       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:25.589284123Z W0109 04:54:25.589246       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:25.640334752Z W0109 04:54:25.640301       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:27.846068983Z W0109 04:54:27.844838       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:36.123897808Z W0109 04:54:36.123854       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:39.102844215Z W0109 04:54:39.102804       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:40.639974153Z W0109 04:54:40.639937       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:40.931479478Z W0109 04:54:40.931437       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:41.878500956Z W0109 04:54:41.878460       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:42.729139410Z W0109 04:54:42.729093       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:42.979441489Z W0109 04:54:42.979401       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:43.088466648Z W0109 04:54:43.088428       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:46.612431962Z W0109 04:54:46.612391       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.648147357Z W0109 04:56:10.648037       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.650560861Z W0109 04:56:10.650529       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.652751815Z W0109 04:56:10.652720       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.652904357Z W0109 04:56:10.652884       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.657924367Z W0109 04:56:10.657896       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.662490705Z W0109 04:56:10.662464       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.662527657Z W0109 04:56:10.662467       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.662823277Z W0109 04:56:10.662797       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.663511065Z W0109 04:56:10.663489       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.649180343Z W0109 04:56:11.649148       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.651700600Z W0109 04:56:11.651670       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.654143551Z W0109 04:56:11.654119       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.654160951Z W0109 04:56:11.654126       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.658636041Z W0109 04:56:11.658616       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.663199958Z W0109 04:56:11.663173       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.663281758Z W0109 04:56:11.663210       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.663281758Z W0109 04:56:11.663220       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.664683370Z W0109 04:56:11.664659       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:12.970360909Z W0109 04:56:12.970316       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.090981278Z W0109 04:56:13.090944       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.091046699Z W0109 04:56:13.091029       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.184632543Z W0109 04:56:13.184592       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.199287128Z W0109 04:56:13.199255       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.204789514Z W0109 04:56:13.204761       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.238835882Z W0109 04:56:13.238800       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.320642806Z W0109 04:56:13.320603       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.520060327Z W0109 04:56:13.520028       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.428161872Z W0109 04:56:15.428121       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.589262412Z W0109 04:56:15.589228       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.736146757Z W0109 04:56:15.736108       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.781757534Z W0109 04:56:15.781715       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.855515979Z W0109 04:56:15.855483       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.893109515Z W0109 04:56:15.893075       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:16.114251089Z W0109 04:56:16.114205       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:16.359860619Z W0109 04:56:16.359821       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:16.568267176Z W0109 04:56:16.568222       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.084668161Z W0109 04:56:19.084621       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.229897158Z W0109 04:56:19.229852       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.511973899Z W0109 04:56:19.511929       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.597853549Z W0109 04:56:19.597817       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.787874687Z W0109 04:56:19.787838       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.187103624Z W0109 04:56:20.187062       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.343333441Z W0109 04:56:20.343294       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.474010401Z W0109 04:56:20.473959       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.758745861Z W0109 04:56:20.758705       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:24.422185143Z W0109 04:56:24.422140       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:25.343260914Z W0109 04:56:25.343223       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:25.794640123Z W0109 04:56:25.794591       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:25.943163595Z W0109 04:56:25.943126       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:26.119266117Z W0109 04:56:26.119227       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:26.226346295Z W0109 04:56:26.226303       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:26.651793237Z W0109 04:56:26.651754       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:28.195628975Z W0109 04:56:28.195588       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:28.298706994Z W0109 04:56:28.298667       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:34.304724057Z W0109 04:56:34.304678       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:34.810313017Z W0109 04:56:34.810275       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:35.946668435Z W0109 04:56:35.946628       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:36.851487033Z W0109 04:56:36.851446       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:37.859643186Z W0109 04:56:37.859603       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:38.656402403Z W0109 04:56:38.656362       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:38.682058315Z W0109 04:56:38.682023       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:39.131596910Z W0109 04:56:39.131557       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:39.305832831Z W0109 04:56:39.305797       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:39.683384983Z W0109 04:56:39.683344       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:40.187018541Z W0109 04:56:40.186968       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:41.003092345Z W0109 04:56:41.003046       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:43.989963127Z W0109 04:56:43.989923       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:47.496218652Z W0109 04:56:47.496176       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:48.379196673Z W0109 04:56:48.379156       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:48.431847017Z W0109 04:56:48.431809       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:51.480059393Z W0109 04:56:51.479981       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:53.105679378Z W0109 04:56:53.105640       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:53.632529513Z W0109 04:56:53.632482       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:53.840553104Z W0109 04:56:53.840516       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:55.321793051Z W0109 04:56:55.321751       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:55.583937398Z W0109 04:56:55.583892       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:57.386148105Z W0109 04:56:57.386109       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:57.504840233Z W0109 04:56:57.504803       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.150632568Z W0109 04:58:03.150549       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.150776015Z W0109 04:58:03.150667       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.150887020Z W0109 04:58:03.150705       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.151243151Z W0109 04:58:03.150822       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.151372398Z W0109 04:58:03.151355       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.151573331Z W0109 04:58:03.151543       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.151804770Z W0109 04:58:03.151776       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.161802983Z W0109 04:58:03.161772       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.162022723Z W0109 04:58:03.161956       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.151238826Z W0109 04:58:04.151203       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.151238826Z W0109 04:58:04.151205       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.151343283Z W0109 04:58:04.151327       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.151435554Z W0109 04:58:04.151417       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.152633953Z W0109 04:58:04.152614       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.152679374Z W0109 04:58:04.152657       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.152691744Z W0109 04:58:04.152684       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.163216923Z W0109 04:58:04.163185       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.163262560Z W0109 04:58:04.163247       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.514447983Z W0109 04:58:05.514412       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.515621840Z W0109 04:58:05.515592       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.569104241Z W0109 04:58:05.569069       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.587392600Z W0109 04:58:05.587362       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.607626092Z W0109 04:58:05.607596       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.612884879Z W0109 04:58:05.612859       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.670937806Z W0109 04:58:05.670901       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.799612563Z W0109 04:58:05.799572       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:06.025581649Z W0109 04:58:06.025546       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:07.674583553Z W0109 04:58:07.674536       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:07.753406061Z W0109 04:58:07.753366       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.106881942Z W0109 04:58:08.106847       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.195436438Z W0109 04:58:08.195393       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.206698123Z W0109 04:58:08.206667       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.508631266Z W0109 04:58:08.508595       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.531129876Z W0109 04:58:08.531082       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.551486157Z W0109 04:58:08.551439       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.617534328Z W0109 04:58:08.617497       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:11.916095909Z W0109 04:58:11.916056       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:11.993477646Z W0109 04:58:11.993442       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.017069677Z W0109 04:58:12.017032       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.041607687Z W0109 04:58:12.041561       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.071900159Z W0109 04:58:12.071865       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.419695152Z W0109 04:58:12.419663       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.855999519Z W0109 04:58:12.855956       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.893276784Z W0109 04:58:12.893240       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:13.122255799Z W0109 04:58:13.122219       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:18.554247157Z W0109 04:58:18.554209       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:18.762538783Z W0109 04:58:18.762500       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.161578707Z W0109 04:58:19.161536       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.279451002Z W0109 04:58:19.279414       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.328874311Z W0109 04:58:19.328836       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.644395442Z W0109 04:58:19.644358       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.743299549Z W0109 04:58:19.743257       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.785835258Z W0109 04:58:19.785801       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.900282145Z W0109 04:58:19.900243       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:27.459099986Z W0109 04:58:27.459060       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:28.942421012Z W0109 04:58:28.942382       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:29.179403970Z W0109 04:58:29.179365       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:29.280370299Z W0109 04:58:29.280333       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:30.669185778Z W0109 04:58:30.669153       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:31.052895115Z W0109 04:58:31.052855       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:31.218570703Z W0109 04:58:31.218533       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:31.563160633Z W0109 04:58:31.563115       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:31.890388173Z W0109 04:58:31.890350       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:34.048973610Z W0109 04:58:34.048932       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:35.049714172Z W0109 04:58:35.049669       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:36.934535173Z W0109 04:58:36.934499       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:39.064984794Z W0109 04:58:39.064942       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:43.174272412Z W0109 04:58:43.174231       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:43.345031673Z W0109 04:58:43.344293       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:44.848282656Z W0109 04:58:44.848244       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:45.171363409Z W0109 04:58:45.171320       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:45.462610621Z W0109 04:58:45.462567       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:45.827184878Z W0109 04:58:45.827147       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:47.307849384Z W0109 04:58:47.307806       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:48.389434231Z W0109 04:58:48.389400       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:48.767350155Z W0109 04:58:48.767316       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:49.225788623Z W0109 04:58:49.225751       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:50.099947794Z W0109 04:58:50.099908       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.680557090Z W0109 05:00:01.680518       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.680966286Z W0109 05:00:01.680940       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.681280927Z W0109 05:00:01.681067       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.681340658Z W0109 05:00:01.681100       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.681340658Z W0109 05:00:01.681144       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.681340658Z W0109 05:00:01.681179       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.681340658Z W0109 05:00:01.681213       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.685306013Z W0109 05:00:01.685280       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.687783426Z W0109 05:00:01.687691       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.682008887Z W0109 05:00:02.681954       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.682732844Z W0109 05:00:02.682696       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.682789995Z W0109 05:00:02.682759       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.682806916Z W0109 05:00:02.682791       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.682806916Z W0109 05:00:02.682760       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.682815273Z W0109 05:00:02.682759       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.682824376Z W0109 05:00:02.682771       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.686682610Z W0109 05:00:02.686663       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.688569049Z W0109 05:00:02.688541       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:03.980699029Z W0109 05:00:03.980657       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.091065739Z W0109 05:00:04.091030       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.141119798Z W0109 05:00:04.141081       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.171256450Z W0109 05:00:04.171217       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.192635514Z W0109 05:00:04.192600       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.218756710Z W0109 05:00:04.218723       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.341244484Z W0109 05:00:04.341200       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.344121961Z W0109 05:00:04.344095       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.455266705Z W0109 05:00:04.455225       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.264914336Z W0109 05:00:06.264871       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.369075195Z W0109 05:00:06.369038       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.708518793Z W0109 05:00:06.708475       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.721670103Z W0109 05:00:06.721637       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.024668825Z W0109 05:00:07.024630       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.090105709Z W0109 05:00:07.090072       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.142113523Z W0109 05:00:07.142079       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.243276286Z W0109 05:00:07.243238       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.386705568Z W0109 05:00:07.386667       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:09.630732756Z W0109 05:00:09.630691       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.568449990Z W0109 05:00:10.568402       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.733574819Z W0109 05:00:10.733533       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.884165423Z W0109 05:00:10.884126       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.972168034Z W0109 05:00:10.972135       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.077542867Z W0109 05:00:11.077507       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.264076788Z W0109 05:00:11.264035       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.269950997Z W0109 05:00:11.269924       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.276828330Z W0109 05:00:11.276798       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.607186319Z W0109 05:00:11.607148       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.885174823Z W0109 05:00:11.885132       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:13.757552146Z W0109 05:00:13.757512       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:15.995321304Z W0109 05:00:15.995277       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:16.516185031Z W0109 05:00:16.516147       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:16.604375815Z W0109 05:00:16.604334       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:16.636421265Z W0109 05:00:16.636382       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:17.139463091Z W0109 05:00:17.139424       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:17.209752684Z W0109 05:00:17.209712       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:17.211578110Z W0109 05:00:17.211553       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:18.602076984Z W0109 05:00:18.602030       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:18.830409890Z W0109 05:00:18.830372       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:19.076117008Z W0109 05:00:19.076080       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:20.313611147Z W0109 05:00:20.313575       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:24.587331487Z W0109 05:00:24.587294       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:25.582360673Z W0109 05:00:25.582325       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:26.236729152Z W0109 05:00:26.236683       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:26.605010591Z W0109 05:00:26.604952       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:26.819417988Z W0109 05:00:26.819377       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:27.538481265Z W0109 05:00:27.538448       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:27.830705258Z W0109 05:00:27.830671       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:28.378454943Z W0109 05:00:28.378417       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:29.126027057Z W0109 05:00:29.125970       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:29.744156552Z W0109 05:00:29.744118       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:42.058054435Z W0109 05:00:42.058006       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:42.689760745Z W0109 05:00:42.689715       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:43.510949844Z W0109 05:00:43.510910       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:44.051254868Z W0109 05:00:44.051211       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:45.600555715Z W0109 05:00:45.600516       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:46.585023187Z W0109 05:00:46.584951       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:47.319111153Z W0109 05:00:47.319069       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:47.824903863Z W0109 05:00:47.824864       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:48.798542710Z W0109 05:00:48.798504       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
